prac_07/project_management.py

```diff
@@ -69,16 +69,34 @@ def display_projects(projects):
             print("\t", project)
 
 
-def filter_projects_by_date():
-    pass
-
-
-def add_new_project():
-    pass
-
-
-def update_project():
-    pass
+def filter_projects_by_date(projects):
+    date = (input("Show projects that start after date (dd/mm/yy): "))
+    date = datetime.datetime.strptime(date, "%d/%m/%Y").date()
+    for project in projects:
+        if project.start_date > date:
```

Comment:
this line results in a TypeError: '>' not supported between instances of 'str' and 'datetime.date'

you may look at comparing strings and datetime